elastic/kibana
# [ResponseOps][Alert Summaries] Add default summary message

Resolves https://github.com/elastic/kibana/issues/148537

## Summary

Added the capability for rule types to provide a default message for summary actions. This default message gets pre-populated within the rule action form whenever a user adds an action to a rule.

Also added a default framework summary message.

### Checklist



### To verify

- Create a rule and add an action and select `notify on custom action intervals`
- Verify that the default message is populated
```
The system has detected {{alerts.new.count}} new, {{alerts.ongoing.count}} ongoing, and {{alerts.recovered.count}} alerts.
```
